projects/plugins/boost/app/modules/cache/pre-wordpress/storage/File_Storage.php

```diff
@@ -115,8 +115,10 @@ public function invalidate( $path, $type ) {
 
 		if ( in_array( $type, array( Boost_Cache_Utils::DELETE_FILES, Boost_Cache_Utils::DELETE_ALL ), true ) && is_dir( $normalized_path ) ) {
 			return Boost_Cache_Utils::delete_directory( $normalized_path, $type );
-		} elseif ( $type === Boost_Cache_Utils::DELETE_FILE ) {
+		} elseif ( $type === Boost_Cache_Utils::DELETE_FILE && is_file( $normalized_path ) ) {
 			return Filesystem_Utils::delete_file( $normalized_path );
+		} else {
+			return true; // no cache file or directory to delete.
```

Comment:
Should this be a wp_error, or false? "invalidate() -> "failed to invalidate" -> returns true 🤔 ? Or am I reading that incorrectly?

Reply:
I want it to return true to mean that there's no cache file to delete/the user doesn't have to worry about cache files.

But maybe WP_Error might be better, so a message can be returned.

Reply:
I agree. `WP_Error` with an adequate message would be ideal in this case.

Reply:
OK, I'll update the PR. Thanks!

Reply:
But, but. I like my `} else {`. :)